### PR TITLE
SumOddAmountEven

### PR DESCRIPTION
**Pull request by Diaboliq**
Given an array of integers, determine the sum of odd numbers and the amount of even numbers

### CODE REVIEW

File: SumOddAmountEven [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            myArray[i] = int_random;
+        }
+
+        System.out.println("Initial values:\n" + Arrays.toString(myArray));
+
+        for (int i=0; i<myArray.length; i++) {
+            if (myArray[i]%2 != 0 ) {
+                sumOddNrs += myArray[i];
+            } else {
+                amountEvenNrs += 1;
```
Comment (CristianManic) [inline SumOddAmountEven:28]: create a method that will return you the amount of even numbers

File: SumOddAmountEven [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        for (int i=0; i<myArray.length; i++) {
+            int int_random = rand.nextInt(upperLimit);
+            myArray[i] = int_random;
+        }
+
+        System.out.println("Initial values:\n" + Arrays.toString(myArray));
+
+        for (int i=0; i<myArray.length; i++) {
+            if (myArray[i]%2 != 0 ) {
+                sumOddNrs += myArray[i];
```
Comment (CristianManic) [inline SumOddAmountEven:26]: create a method that will return you the sum of odd numbers

File: SumOddAmountEven [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        int[] myArray = new int[10];
+        int sumOddNrs = 0;
+        int amountEvenNrs = 0;
+
+        Random rand = new Random();
+        int upperLimit = 1000;
+
+        for (int i=0; i<myArray.length; i++) {
+            int int_random = rand.nextInt(upperLimit);
+            myArray[i] = int_random;
```
Comment (CristianManic) [inline SumOddAmountEven:19]: extract this generation on population of array into separate method

File: SumOddAmountEven (hunk 1 above)
Comment (Diaboliq) [reply SumOddAmountEven:28]: Done

Comment (Diaboliq) [reply SumOddAmountEven:28]: and the method for sum of odd numbers extracted too

File: SumOddAmountEven (hunk 3 above)
Comment (Diaboliq) [reply SumOddAmountEven:19]: Done

File: SumOddAmountEven (hunk 2 above)
Comment (Diaboliq) [reply SumOddAmountEven:26]: Done

**Pull request merged**